sklearn/model_selection/tests/test_validation.py

```diff
@@ -921,7 +921,7 @@ def check_cross_val_predict_with_method(est):
     X, y = shuffle(X, y, random_state=0)
     classes = len(set(y))
 
-    kfold = KFold(len(iris.target))
```

Comment:
Yes, this looks like an error in moving to the recent model_selection restructure